public/components/common/charts/visualizations/basic.tsx

```diff
@@ -34,51 +36,55 @@ export const VisualizationBasic = ({
   noDataMessage,
   errorTitle = 'Error',
   errorMessage,
-  error
+  error,
+  select
```

Comment:
**nitpick** If we want to include some element before the legend, we should not call it `select` and I would suggest a more generic name.